code/socialDistribution/views.py

```diff
@@ -842,7 +843,12 @@ def post_comment(request, author_id, post_id):
                     "object": post.public_id
                 }
 
-                request_url = f'{post.author.strip("/")}/inbox/'
+                if url_parser.get_host(post.public_id) == REMOTE_NODES["t16"]:
```

Comment:
Thoughts on making this the default URL, not just for T16? It looks like T08 and T11 also use the /comments endpoint for posting new comments.

It looks like it's just our group that posts comments to /inbox

https://github.com/DecKirin/404_group_project_21F/wiki/API-documentation#comments

https://github.com/CMPUT404-F21T0/CMPUT404-Project-BetterSocial/wiki/API-Documentation#comments